Makefile

```diff
@@ -4,6 +4,22 @@ SCSS_FILES := $(shell find 'meinberlin/assets/scss' -name '*.scss')
 JS_FILES := $(shell find 'meinberlin/assets/js' | grep '\.jsx\?$$')
 PO_FILES := $(shell find . -name '*.po')
 
+help:
+	@echo mein.berlin development tools
+	@echo
+	@echo It will either use a exisiting virtualenv if it was entered
+	@echo before or create a new one in the .env subdirectory.
+	@echo
+	@echo usage:
+	@echo
+	@echo   make install      -- install dev setup
+	@echo   make build        -- build js and css and create new po and mo files
+	@echo   make lint         -- lint javascript and python
+	@echo   make server       -- start a dev server
+	@echo   make watch        -- start a dev server and rebuild js and css files\
+	                             on changes
+	@echo
```

Comment:
The formatting is lost when I run `make`. I think wrapping it in quotes would preserve spaces.